Makefile.am

```diff
@@ -72,6 +72,6 @@ CLEANFILES += $(CXX_PROTOS)
 	$(PROTOC) $(PROTOCFLAGS) --cpp_out=isolator/. $^
 
 # Library containing kerberos ticket forwarding module.
-pkglib_LTLIBRARIES += libcalico_isolator.la
-libcalico_isolator_la_SOURCES = isolator/network_isolator.cpp ${CXX_PROTOS}
-libcalico_isolator_la_LDFLAGS = -release $(PACKAGE_VERSION) -shared $(MESOS_LDFLAGS)
+pkglib_LTLIBRARIES += libnetwork_isolator.la
```

Comment:
How about libmesos_network_isolator instead? That way it's easier to know that the library is for mesos.